blueprints/frost-component/files/__root__/__path__/__name__.js

```diff
@@ -1,13 +1,34 @@
 import Ember from 'ember'
-import FrostEvents from 'ember-frost-core/mixins/frost-events'
-import {
-  PropTypes
-} from 'ember-prop-types'
+const { Component } = Ember
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
+import layout from '<%= templatePath %>'
 
-const {
-  Component
-} = Ember
+const <%= classifiedModuleName %> = Component.extend(PropTypesMixin, {
```

Comment:
I really should use the word "route" when taking ember b/c clearly I didn't mean an ember route ;)